### ISSUE TITLE
[Bug Report][3.9.5] VForm A11y issue:  fast-fail inputs will not be announced to screenreader users when submitting invalid forms

### ISSUE THREAD

**Issue by NFriedo** [vuetifyjs/vuetify#21920]
### Environment
**Vuetify Version:** 3.9.5
**Vue Version:** 3.5.18
**Browsers:** Chrome 139.0.0.0, Safari
**OS:** Mac OS 10.15.7

### Steps to reproduce
Tested with VoiceOver on Safari/Chrome
1. Navigate to the submit button without filling out the required fields
2. Click the submit button

### Expected Behavior
Error messages are announced after activating the button

### Actual Behavior
No error messages are announced, probably because the aria live region is not updated and focus does not change

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNqVU02PmzAQ/SsjX5JIAbKL9rAooe2lp6qH9hhSycCwsWoMssdpqlX+e22TQKJqtVmf5uN5mPce3r4yo6vkS9/HB4ssY2vCtpecMC8UwPoQmT0iQSW5MZuCtceIW+oKBn9ETXtXSVerggVwgDedbqHhhqKGCwmfjS1bQXGv8YCKzriAJDw6kEBZX4oAh6jtapRubCO0oe+8xYJN7UxbiX6NsfvDF64hkpfh/lePAHUzIF8n1591y9y9jqP/9jaX5pvLfHOAj+xSkpoUp+jRyU1/e3TZIKfLS9lVv/OfIfWj3JWLCS7zLgwGuiQ46LJ1cuWtS02lRT9Ygse+0wQ1NtxKgtdhUs2JZzBfwCaH+bkGMGqfwWy2/K8aRMhge6Uilxb9jHGEP6KBeeh8iiWqF9pDvoF0ARrJagWkLU6a+HNuzCZnobUuKhE4gUSvcQrVnmteEWoTz6bbp3HP3RhdXHM0Hh7TicmNm3cSSba/VtHzLokJDQ20FndQGf8LqLgC1bl31iniQkEtXgS9Q+G0CMHJOzt6yU5LlsZP8UPKfPAcP7Flw6XB5c1DH2q7f/3VNg4=)

### Other comments
Maybe one way to fix this issue would be to set focus to the first invalid input, which would announce the error that describes the input again.
